cores/esp8266/spiffs_api.h

```diff
@@ -124,6 +127,27 @@ class SPIFFSImpl : public FSImpl
         return true;
     }
 
+    bool mkdir(const char* path) override
+    {
+        (void)path;
+        return false;
+    }
+
+    bool rmdir(const char* path) override
+    {
+        (void)path;
+        return false;
+    }
+
+    bool setConfig(const FSConfig *cfg) override
```

Comment:
How about passing by const reference instead of by const pointer? No nullptr allowed.

Reply:
Good idea, makes nicer code.  Updated in latest push.